eslint/babel-eslint-parser/src/worker/asyncMaybeParse.cts

```diff
@@ -0,0 +1,59 @@
+
+import babel = require("./babel-core.cts");
```

Comment:
Thank you for the PR. The `babel-core.cts` is a subset of `@babel/core`, so you have to update that as well.